src/main/java/nextstep/infrastructure/AuthorizationExtractor.java

```diff
@@ -0,0 +1,28 @@
+package nextstep.infrastructure;
+
+import lombok.RequiredArgsConstructor;
+import org.apache.logging.log4j.util.Strings;
+import org.springframework.stereotype.Component;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.Enumeration;
+
+@Component
+@RequiredArgsConstructor
+public class AuthorizationExtractor {
+    private static final String AUTHORIZATION = "Authorization";
+    private static final String ACCESS_TOKEN_TYPE = "accessTokenType";
+    public static String BEARER_TYPE = "Bearer";
+
+    public String extract(HttpServletRequest request) {
+        Enumeration<String> headers = request.getHeaders(AUTHORIZATION);
+        while (headers.hasMoreElements()) {
+            String value = headers.nextElement();
+            if ((value.toLowerCase().startsWith(BEARER_TYPE.toLowerCase()))) {
```

Comment:
```suggestion
            if (isBearerType(value)) {
```
함수 추출을 하면 코드 이해가 더 편할거같아요~!

Reply:
훨씬낫겠네요. 변경하겠습니다!